CMakeLists.txt

```diff
@@ -55,7 +55,8 @@ add_definitions(-fPIC)
 
 # Baseline compiler flags, any change here will affect all build types
 #set (CMAKE_CXX_FLAGS "-Wall -Wno-unknown-pragmas -D_LARGEFILE64_SOURCE -D_GNU_SOURCE -Werror")
-set (CMAKE_CXX_FLAGS "-Wall -Wno-unknown-pragmas -D_LARGEFILE64_SOURCE -D_GNU_SOURCE -Werror -fno-var-tracking-assignments")
+set (CMAKE_CXX_FLAGS "-Wall -Wno-unknown-pragmas -D_LARGEFILE64_SOURCE -D_GNU_SOURCE -fno-var-tracking-assignments -I${CMAKE_INCLUDE_PATH}")
```

Comment:
This definitely doesn't compile with modern g++ (version 7 in particular) - it complains a lot about auto_ptr being deprecated. As I didn't find any suggestion about gcc version, I found it easier just to remove this flag. But I agree that this is bad. 

I'll revert this line in particular and check which compiler it is used to build Orion using the dockerfile (the default gcc package with no mention to which version)